tools/create_packages.sh

```diff
@@ -3,80 +3,73 @@
 set -e
 
 echo "Packaging"
+echo "---------"
 
-# This creates a version such as "v1.2.3-5-g123abc".
-version=`git describe --always --tags`
-# We want to extract 1.2.3 from it.
-version=`echo ${version} | sed 's/v\([0-9]*\.[0-9]*\.[0-9]*\).*$/\1/'`
+if [ "$#" -ne 2 ]; then
+  echo "Error: missing path to the 'install' directory (i.e. what was specified as -DCMAKE_INSTALL_PREFIX)" >&2
+  echo "       or output path (where the package will be generated)!" >&2
+  echo "Usage: $0 <path/to/install> <output/path>" >&2
+  exit 1
+fi
 
 script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-working_dir=${script_dir}/install
-install_dir=${working_dir}/usr
+install_dir=$1
+output_dir=$2
 
-rm -rf ${install_dir}
-
-cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=${install_dir} -Bbuild/release-package -H.;
-make -Cbuild/release-package install -j4;
-
-common_args="--input-type dir  \
-    --version ${version} \
-    --maintainer julian@oes.ch \
-    --url https://sdk.dronecode.org \
-    --license BSD-3-Clause \
-    --force"
-
-pushd ${working_dir}
+working_dir=$(mktemp -d)
+mkdir -p ${working_dir}/install
+cp -r ${install_dir} ${working_dir}/install/usr
 
 # We need the relative path of all files.
-library_files=`find . -type f | cut -sd / -f 2-`
+library_files=`find ${working_dir}/install -type f | cut -sd / -f 5-`
+echo "Files to be packaged:"
 echo ${library_files}
 
-echo "#!/bin/sh" > run_ldconfig
+# This creates a version such as "v1.2.3-5-g123abc".
+version=`git describe --always --tags`
+# We want to extract 1.2.3 from it.
+version=`echo ${version} | sed 's/v\([0-9]*\.[0-9]*\.[0-9]*\).*$/\1/'`
+
+echo "#!/bin/sh" > ${working_dir}/run_ldconfig
+echo "/sbin/ldconfig" >> ${working_dir}/run_ldconfig
+
+common_args="--chdir ${working_dir}/install \
+             --input-type dir  \
+             --name mavsdk \
+             --version ${version} \
+             --maintainer julian@oes.ch \
+             --url https://sdk.dronecode.org \
```

Comment:
This one is probably going to change, right?

Reply:
Yes, I guess as soon as we have the new domain :+1: